jobs/BuildBaseImage/base_docker/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && apt-get install -y \
+  wget \
+  curl \
+  vim \
+  apt-transport-https \
+  && rm -rf /var/lib/apt/lists/*
+
+# nodejs
+RUN curl --silent https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add - \
+  && echo "deb https://deb.nodesource.com/node_4.x trusty main" | tee /etc/apt/sources.list.d/nodesource.list \
+  && echo "deb-src https://deb.nodesource.com/node_4.x trusty main" | tee -a /etc/apt/sources.list.d/nodesource.list \
+  && apt-get update \
+  && apt-get install -y nodejs \
+  && rm -rf /var/lib/apt/lists/*
+
+# pm2
+RUN npm install -g pm2
+
+# prerequisite packages of RackHD
+RUN apt-get update && apt-get install -y \
+  mongodb \
+  rabbitmq-server \
+  isc-dhcp-server \
```

Comment:
[non-blocking-comment] I think we need to explicitly specific the version of 3rd party tools eventually,.